classes/components/listPanels/PKPSelectReviewerListPanel.inc.php

```diff
@@ -62,6 +62,15 @@ public function getConfig() {
 						// The slider component expects variables in the format {var}
 						'formatter' => str_replace('$', '', __('common.moreThan')),
 					],
+					[
+						'param' => 'reviewsCancelled',
+						'title' => __('reviewer.list.cancelledReviews'),
+						'value' => 10,
+						'min' => 0,
+						'max' => 20,
+						// The slider component expects variables in the format {var}
+						'formatter' => str_replace('$', '', __('common.moreThan')),
+					],
```

Comment:
I think we can leave this filter out. We also don't provide a filter for declined reviews, and I think this would be a rarely used filter. Better to leave it out so that users can find more useful filters more quickly.

Reply:
Removed.